interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -373,6 +357,20 @@ protected final ClientInterceptor createCensusStatsClientInterceptor() {
                 true, true, true, false /* real-time metrics */);
   }
 
+  protected final ServerStreamTracer.Factory createCustomCensusTracerFactory() {
+    return InternalCensusStatsAccessor.getServerStreamTracerFactory(
+        tagger, tagContextBinarySerializer, serverStatsRecorder,
+        GrpcUtil.STOPWATCH_SUPPLIER,
+        true, true, true, false /* real-time metrics */);
+  }
+
+  /**
+   * Return {@code true} when custom census module used.
+   */
+  protected boolean customCensusModulePresent() {
+    return false;
```

Comment:
Seems this was previously tangentally linked to `metricsExpected()`. Could we default to returning the value of `metricsExpected()`? I think very few tests would have the values be different, maybe only InProcessTest?